README.rst

```diff
@@ -36,7 +44,11 @@ Build with cmake from the host_xvf_control/ folder:
 
 .. note::
 
-    Windows drivers can only be built with 32-bit tools
+    Windows drivers can only be built with 32-bit tools.
```

Comment:
This is sad :(

Reply:
It is not a big problem to build 64-bit host apps, the only downside is maintenance. The advantage of using 32-bit versions is that they work both on 32 and 64 bit platforms. Unfortunately the 64 bit versions don't work on 32-bit platform.

We  can look into support both 32 and 64 bit versions in the future, it is not very complicated to do.